### DISCUSSION TITLE
simplyfy invalid user account dialog

### DISCUSSION THREAD

**Discussion by peter1384** [BearWare/TeamTalk5#2343] [Ideas]
so now that connect to a server dialog, user accounts dialog, command flud dialog has been simpled, there is one more.
Whenever you try to loggin to a server with an invalid username or invalid password, It doesn't matter if both username and password are incorrect, or only one of them username or password is incorrect, we are asked to enter both of them.
That is, entering username and pressing the ok button, and then the second dialog opens, for entering the password
This dialog should be simpled with a username edit box, a password edit box, an ok and a cansle button, also a "show password" checkbox If agreed.
This allowes to shift tab to correct prev entrys, and also simples the dialog, like the others.

**Comment by peter1384** [comment]
@CoBC If you want, Implement this, otherwise i'll close this to prevent so many open ideas, also i'll close all my other ideas.
Thanks

**Comment by amirmahdifard** [comment]
at some point, in server list dialog, a show password checkbox has been added for both channel password and user password, but now when we press enter on a channel to join it, We also need this checkbox there also, because at some points we need to see the things that we are pressing, so I hope this agreed

**Comment by peter1384** [comment]
@CoBC also, like user info dialog, the read only edit boxes in server stats dialog could be replaced by q static lables. Also, We have an unlabled spin button for screenreaders when the update intevul checkbox is checked, So that should be labled as wel for improoving accessibility
